spec/acceptance/attributes_for_spec.rb

```diff
@@ -1,3 +1,7 @@
+if RUBY_VERSION >= "3.0" && RUBY_ENGINE != "truffleruby"
```

Comment:
@mike-burns CI forced me to skip Truffle Ruby. Since this is only introducing test coverage, one that implementation adopts the latest language features support will kick-in without additional implementation changes on our end.